crypto/signer.go

```diff
@@ -83,10 +83,9 @@ func (s *Signer) SignObject(obj interface{}) (*sigpb.DigitallySigned, error) {
 	return s.Sign(hash[:])
 }
 
-// SignLogRoot hashes and signs the supplied (to-be) SignedLogRoot and returns a
-// signature.  Hashing is performed by github.com/benlaurie/objecthash.
-func (s *Signer) SignLogRoot(root *trillian.SignedLogRoot) (*sigpb.DigitallySigned, error) {
-	hash, err := HashLogRoot(*root)
+// SignLogRoot hashes and signs the supplied (to-be) SignedLogRoot and returns a SignedLogRoot
```

Comment:
Maybe "returns a complete SignedLogRoot (including signature)". 

(also nit) Trailing full stop in comment.

Reply:
Done